submissions/victorAienobe/database/init.sql

```diff
@@ -0,0 +1,130 @@
+-- Created database for my e-commerce store
+CREATE DATABASE e_commerce_store;
+USE e_commerce_store;
+
+-- Created the categories table
+CREATE TABLE categories (
+	id INT NOT NULL AUTO_INCREMENT,
+	category_name VARCHAR(20) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+-- Created the admins table
+CREATE TABLE admins (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(50) NOT NULL,
+	email VARCHAR(50) NOT NULL,
+	password VARCHAR(15) NOT NULL,
+	phone_number VARCHAR(15) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+-- created the products table
+CREATE TABLE products (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(50) NOT NULL,
+	description VARCHAR(250) NOT NULL,
+	image BLOB,
+    category_id INT,
+    admin_id INT,
+	unit_price INT NOT NULL,
+    stock_level INT NOT NULL,
+    status VARCHAR(20) NOT NULL,
+	PRIMARY KEY (id),
+    FOREIGN KEY (category_id) REFERENCES categories (id) ON DELETE CASCADE,
+    FOREIGN KEY (admin_id) REFERENCES admins (id) ON DELETE CASCADE
+);
+
+-- Created the customers table
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	email_address VARCHAR(40) NOT NULL,
+	password VARCHAR(15) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+-- Created the customer addresses table
+CREATE TABLE customer_addresses (
+	id INT NOT NULL AUTO_INCREMENT,
+	customer_id INT,
+	street_address VARCHAR(100) NOT NULL,
+	city VARCHAR(20) NOT NULL,
+	state VARCHAR(20) NOT NULL,
+	zip_code INT,
+	country VARCHAR(20) NOT NULL,
+	phone_number BIGINT NOT NULL,
+    PRIMARY KEY (id),
+	FOREIGN KEY (customer_id) REFERENCES customers (id) ON DELETE CASCADE
+);
+
+-- Created the orders table
+CREATE TABLE orders (
+	id INT NOT NULL AUTO_INCREMENT,
+	customer_orders INT,
+	order_amount DECIMAL(7,2) NOT NULL,
+	order_created_at DATETIME,
+	PRIMARY KEY (id),
+	FOREIGN KEY (customer_orders) REFERENCES customers (id) ON DELETE CASCADE
+);
+-- Auto increment count is set to start from 100
+ALTER TABLE orders AUTO_INCREMENT = 100;
+
+-- Created the order items table
+CREATE TABLE order_items (
+	id INT NOT NULL AUTO_INCREMENT,
+	order_id INT,
```

Comment:
should be `NOT NULL`. or is there a reason why you want it to be `NULL`. cos by default, that is what it is